package.json

```diff
@@ -18,6 +18,7 @@
     "htmlhint": "~0.9.13",
     "jshint": "~2",
     "postcss-cli": "~2",
+    "prettier": "^1.9.2",
```

Comment:
IMO you need to add this is in package.json as well

Reply:
It is in package.json

Reply:
Sorry @newbazz i could not understand 
'This is in package.json'
Can you please elaborate??

Reply:
@Nshul Hey, I looked at the Circle CI build. It fails at the point where it compares the requirements for prettier. package.json says that it should be "^1.9.2" , but in the linter file you mention in the requirements, NpmRequirement('prettier', '1') . I am not sure if the 1 denotes the version here.
If it does, maybe making it same at both places might work. Maybe it would work.

Reply:
@Nshul @parimatrix is correct change the prettier verison in PrettierBear file.

Reply:
cool I will change it. Thanks @parimatrix @newbazz